protocol-units/da-sequencer/node/src/server.rs

```diff
@@ -15,17 +18,17 @@ use tonic::transport::Server;
 pub async fn run_server(
 	address: SocketAddr,
 	request_tx: mpsc::Sender<GrpcRequests>,
+	whitelist: Arc<RwLock<Whitelist>>,
```

Comment:
Don't need to pass the Arc Mutex, just pass the white list then create the mutex inside the function.
Mutex is specific to the function implementation.